tests/wsgi_test.py

```diff
@@ -2058,7 +2074,7 @@ def ping(self, fd):
     def test_short_read_with_content_length(self):
         body = self.body()
         req = "POST /short-read HTTP/1.1\r\ntransfer-encoding: Chunked\r\n" \
-              "Content-Length:1000\r\n\r\n" + body
```

Comment:
Removed to do not reflect and encourage bad practices, I wonder if this test itself could be removed.
Another option, would be to remove the `transfer-encoding` and rename the test and keep it alive.

Any opinion?